econet_base.yaml

```diff
@@ -154,6 +154,32 @@ text_sensor:
     request_once: true
     icon: "mdi:information-box"
     entity_category: "diagnostic"
+  - platform: template
+    name: "Microcontroller Datetime"
+    id: irtc_time
+    icon: mdi:calendar-clock
+    update_interval: 30s
+    entity_category: "diagnostic"
+    disabled_by_default: true
+    lambda: |-
+      const int timeStringSize = 21;
```

Comment:
I don't think the camel case is compliant to our formatting. Can you temporarily add all this block of code in econet.cpp, run the formatter and then move the code back here?

Reply:
Is snake-case preferred for our C code?

Reply:
We should use https://github.com/esphome/esphome/blob/dev/.clang-tidy